sonar/heg/serializers/schemas/crossref.py

```diff
@@ -154,8 +158,10 @@ def get_part_of(self, obj):
         for issn_type in obj.get('issn-type', []):
             if issn_type['type'] == 'electronic':
                 part_of['document']['identifiedBy'] = [{
-                    'type': 'bf:Issn',
-                    'value': issn_type['value']
+                    'type':
+                    'bf:Issn',
+                    'value':
+                    issn_type['value']
```

Comment:
Cosmetic : why split key/value on two lines ?

Reply:
I hate this, too.. But my python formatter in Visual Studio does not agree..